app/models/section.rb

```diff
@@ -102,6 +109,10 @@ def self.from_publishing_api(content_id:)
     section
   end
 
+  def validate_manual(manual_content_id)
+    raise ManualSection::RecordNotFound if self.manual_content_id != manual_content_id
```

Comment:
I don't think there's a need to raise a different type of error here.  I consider this as being like what ActiveRecord models does when doing a find on a scoped relation; it raises `ActiveRecord::RecordNotFound` regardless of the specific reason it can't be found (e.g. `Post.find(12)` vs. `Post.where(author_id: Author.first).find(12)` - both would raise `ActiveRecord::RecordNotFound`).  

ps. I think I said `ManualSection::RecordNotFound` in my previous review, but that's just because I forgot these were just called `Section`, sorry!